sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/queue/support/ServiceBusQueueTestOperation.java

```diff
@@ -30,7 +31,7 @@
     private final Multimap<String, IMessageHandler> handlersByQueue = ArrayListMultimap.create();
 
     public ServiceBusQueueTestOperation(ServiceBusQueueClientFactory clientFactory) {
```

Comment:
Can we move this class into `src/test/resource`? Because it's only used in test.

Reply:
These two classes are also used in external classes.